macros/InterfaceOverview.ejs

```diff
@@ -36,11 +36,6 @@ if (slug) {
     // Fetch the databases
 
     var webAPIData = string.deserialize(await template('InterfaceData'));
-
-    if (group) {
-        var webAPIGroups = string.deserialize(await template('GroupData'));
-    }
```

Comment:
The easiest thing to do might be to leave all of this group-related code in the macro for now and just focus on the dl fix and the utils.js improvements. I won't be able to sign off on a patch that removes the group stuff without input from the writers who wrote and use the macro.

Reply:
So, I created `InterfaceOverview` originally and generally am the one who has been working on it. I have plans to make further improvements and add more features but have been stalled because of the problems I've had trying to get working tests in place. Once tests are in, I can get back to work on this.

Ideally, this PR will simply fix the missing `</dl>` and add the needed tests. I can then follow up by making the not insignificant code changes I have in mind.

Reply:
I’ve removed the currently unused dependency on `GroupData` to unblock work on https://github.com/mdn/sprints/issues/906, which was requested by @chrisdavidmills.

Reply:
I propose that you leave the group-related lines of code in, but comment them out. Since they currently have no effect, it won't change anything. But by not deleting them entireliy it makes it easier for Sheppy to come back to this macro and resume work on it, and it means you don't have to think about whether to delete the macro argument entirely.